src/data/nat/order/basic.lean

```diff
@@ -322,7 +312,7 @@ proved above, and some of the results in later sections depend on the definition
 lemma diag_induction (P : ℕ → ℕ → Prop) (ha : ∀ a, P (a + 1) (a + 1)) (hb : ∀ b, P 0 (b + 1))
   (hd : ∀ a b, a < b → P (a + 1) b → P a (b + 1) → P (a + 1) (b + 1)) :
   ∀ a b, a < b → P a b
-| 0 (b + 1) h := hb _
+| 0 (b + 1) h       := hb _
```

Comment:
I'm not sure this is an improvement

Reply:
Just to align the `:=` like many other files

Reply:
Yeah but many other files don't.